src/utils.jl

```diff
@@ -0,0 +1,115 @@
+using HTTP
+using JSON3
+using Dates
+
```

Comment:
These packages can be put in the main src file `src/OHDSIAPI.jl`. We generally do that for organization purposes.